test/tools/ossfuzz/protomutators/YulProtoMutator.cpp

```diff
@@ -46,6 +46,34 @@ static LPMPostProcessor<Block> addStoreToZero(
 	}
 );
 
+
+namespace
+{
+template <typename T>
+struct addControlFlow
+{
+	addControlFlow()
+	{
+		function = [](T* _message, unsigned)
+		{
+			MutationInfo m{_message, "Added control flow."};
```

Comment:
Where does this message get displayed?

Reply:
On `std::cout`. I think this needs some refactoring so that debug can be opt in and not part of the fuzzing process. Will make a small PR for it.